src/constants/strings.ts

```diff
@@ -9,7 +9,7 @@ export const WORD_NOT_FOUND_MESSAGE = 'Word not found'
 export const HARD_MODE_ALERT_MESSAGE =
   'Hard Mode can be enabled only at the start!'
 export const HARD_MODE_DESCRIPTION =
-  'Any revealed hints must be used in subsequent guesses'
+  '"In hard mode, tile colors remain neutral, offering no feedback on correct or incorrect guesses." '
```

Comment:
We dont want extra quotes in here

```suggestion
  'In hard mode, tile colors remain neutral, offering no feedback on correct or incorrect guesses.'
```